bluebutton-data-pipeline-rif-extract/src/main/java/gov/hhs/cms/bluebutton/datapipeline/rif/extract/RifFilesProcessor.java

```diff
@@ -290,7 +290,27 @@ private static BeneficiaryRow buildBeneficiaryEvent(CSVRecord csvRecord) {
 		beneficiaryRow.stateCode = csvRecord.get(BeneficiaryRow.Column.STATE_CODE.ordinal());
 		beneficiaryRow.countyCode = csvRecord.get(BeneficiaryRow.Column.BENE_COUNTY_CD.ordinal());
 		beneficiaryRow.postalCode = csvRecord.get(BeneficiaryRow.Column.BENE_ZIP_CD.ordinal());
-		// TODO finish mapping the rest of the columns
+		beneficiaryRow.birthDate = LocalDate.parse(csvRecord.get(BeneficiaryRow.Column.BENE_BIRTH_DT.ordinal()),
+				RIF_DATE_FORMATTER);
+		beneficiaryRow.sex = csvRecord.get(BeneficiaryRow.Column.BENE_SEX_IDENT_CD.ordinal()).charAt(0);
```

Comment:
Please use the `parseCharacter(...)` method in this class, instead. (It should eventually end up with some data-validation logic that we'll want applied to every one of these.)

(For this field and all of the other `char` fields that you're parsing.